public/controllers/dev-tools/dev-tools.js

```diff
@@ -367,7 +367,7 @@ export class DevToolsController {
     const currentState = AppState.getCurrentDevTools();
     if (!currentState) {
       const demoStr =
-        'GET /agents?status=active\n\n#Example comment\nGET /manager/info\n\nGET /syscollector/000/packages?search=ssh&limit=1\n\nPOST /agents\n' +
+        'GET /agents?status=active\n\n#Example comment\n\n#You can use ? if you need hints \n#for your query\n\nGET /manager/info\n\nGET /syscollector/000/packages?search=ssh&limit=1\n\nPOST /agents\n' +
```

Comment:
I don't understand what this message means. 
does It mean that if I add `?` in any place the suggestions will be shown? cause this happens  only when I add the first `?`